src/madsci_workcell_manager/madsci/workcell_manager/workcell_engine.py

```diff
@@ -197,27 +203,30 @@ def handle_data_and_files(
                     label = data_key
                 datapoint = ValueDataPoint(
                     label=label,
-                    step_id=step.id,
+                    step_id=step.step_id,
                     workflow_id=wf.workflow_id,
-                    experiment_id=wf.experiment_id,
+                    experiment_id=wf.ownership_info.experiment_id,
                     value=response.data[data_key],
                 )
                 self.data_client.submit_datapoint(datapoint)
-                labeled_data[label] = datapoint.id
+                labeled_data[label] = datapoint.datapoint_id
         if response.files:
             for file_key in response.files:
                 if step.data_labels is not None and file_key in step.data_labels:
                     label = step.data_labels[file_key]
                 else:
                     label = file_key
                 datapoint = FileDataPoint(
-                    step_id=step.id,
+                    step_id=step.step_id,
                     workflow_id=wf.workflow_id,
-                    experiment_id=wf.experiment_id,
+                    experiment_id=wf.ownership_info.experiment_id,
                     label=label,
                     path=str(response.files[file_key]),
                 )
-                self.data_client.submit_datapoint(datapoint, datapoint.path)
-                labeled_data[label] = datapoint.id
+                default_logger.log_info("presub")
```

Comment:
If we're going to include these little one-off debugging statements, they should be `log_debug` instead of `log_info`